src/modules/modules.controller.ts

```diff
@@ -51,17 +56,20 @@ export class ModulesController {
     return await this.modulesService.getAllByUser(req.user.id);
   }
 
+  @UserTypes(UserEnum.ADMIN)
   @Delete('/:id')
   async delete(@Request() req: RequestWithUser, @Param('id') id: number): Promise<void> {
     await this.modulesService.delete(id);
   }
 
+  @UserTypes(UserEnum.TEACHER, UserEnum.TEACHING_STUDENT)
```

Comment:
add admin

Reply:
Admin has always rights, so no need to add him here